deploy.py

```diff
@@ -43,11 +43,28 @@ def ensure_folder(name, uid, api):
 
 
 
-def build_dashboard(dashboard_path):
-    return json.loads(subprocess.check_output([
-        'jsonnet', '-J', 'vendor',
-        dashboard_path
-    ]).decode())
+def build_dashboard(dashboard_path, api, global_dash=False):
+    cmd = ["jsonnet", "-J", "vendor", dashboard_path]
+
+    # If we need to build a global dashboard across more datasource
+    # pass the list of datasources to be used in the queries
+    if global_dash:
```

Comment:
I'd love to completely get rid of this special casing! Sorry if I wasn't super clear in https://github.com/jupyterhub/grafana-dashboards/pull/39#discussion_r897847029. What I'd love to do is:

1. No specialcasing
2. Next to the 'dashboards/' directory, we have a `global-dashboards/` directory. By default, we only deploy the `dashboards/` directory
3. *If* on a particular grafana someone wants global-dashboards specifically, they can run `deploy.py` twice - once against `dashboards/` and once against `global-dashboards/` to get that. 

How does that sound?

Reply:
Hmm, if I understand correctly, the global-dashboards will get installed only if this directory is explicity deployed by a human?

I'm not sure it's not possible without special casing in the deployer, because this "global" dashboard needs to be passed the list of datasources in the cmd like so:

```
datasources_names = [ds["name"] for ds in datasources]
cmd.extend(["--tla-code", f"datasources={datasources_names}"])
```